addons/mail/static/src/core/common/message.js

```diff
@@ -108,6 +108,7 @@ export class Message extends Component {
         /** @type {ShadowRoot} */
         this.shadowRoot;
         this.root = useRef("root");
+        this.props.forwardRefToParent?.(this.props.message, this.root);
```

Comment:
We're currently not removing "dead" ref (component destroyed), I guess it's not a big deal but it might be cleaner to handle it.